armory/baseline_models/pytorch/yolov3.py

```diff
@@ -0,0 +1,49 @@
+from typing import Optional
+
+import torch
+
+from art.estimators.object_detection import PyTorchYolo
+from pytorchyolo.utils.loss import compute_loss
+from pytorchyolo.models import load_model
+
+
+DEVICE = torch.device("cuda" if torch.cuda.is_available() else "cpu")
+
+
+class Yolo(torch.nn.Module):
+    def __init__(self, model):
+        super().__init__()
+        self.model = model
+
+    def forward(self, x, targets=None):
+        if self.training:
+            outputs = self.model(x)
+            loss, _ = compute_loss(outputs, targets, self.model)
+            loss_components_dict = {"loss_total": loss}
+            return loss_components_dict
+        else:
+            out = self.model(x)
+            return out
+
+
+def get_art_model(
+    model_kwargs: dict, wrapper_kwargs: dict, weights_path: Optional[str] = None
+) -> PyTorchYolo:
+
+    model_path = "armory/baseline_models/pytorch/yolov3.cfg"
+    model = load_model(model_path=model_path, weights_path=weights_path)
+    model_wrapper = Yolo(model)
+
+    params = [p for p in model_wrapper.parameters() if p.requires_grad]
+    optimizer = torch.optim.SGD(params, lr=0.01, momentum=0.9, nesterov=True)
+
+    detector = PyTorchYolo(
+        model=model_wrapper,
+        device_type=DEVICE,
+        input_shape=(416, 416, 3),
+        optimizer=optimizer,
+        clip_values=(0, 1),
+        channels_first=False,
+        attack_losses=("loss_total",),
+    )
+    return detector
```

Comment:
You aren't using `model_kwargs` or `wrapper_kwargs` at all here. I'm not sure what _should_ be kwarg'able but these options seem appropriate to me:

```suggestion
def get_art_model(
    model_kwargs: dict, wrapper_kwargs: dict, weights_path: Optional[str] = None
) -> PyTorchYolo:

    model_path = "armory/baseline_models/pytorch/yolov3.cfg"
    model = load_model(model_path=model_path, weights_path=weights_path)
    model_wrapper = Yolo(model)

    params = [p for p in model_wrapper.parameters() if p.requires_grad]
    optimizer = torch.optim.SGD(
        params,
        lr=model_kwargs.get("learning_rate", 0.01),
        momentum=model_kwargs.get("momentum", 0.9),
        nesterov=model_kwargs.get("nesterov", True),
    )

    detector = PyTorchYolo(
        model=model_wrapper,
        device_type=DEVICE,
        input_shape=(416, 416, 3),
        optimizer=optimizer,
        clip_values=(0, 1),
        channels_first=False,
        attack_losses=wrapper_kwargs.get("attack_losses", ("loss_total",)),
    )
    return detector
```

Reply:
Actually those dicts are supposed to be passed through to load_model and PytorchYolo respectively; thanks for catching that.  In fact I will use model_kwargs to pass the path to the config file instead of hard coding it there.